docs/connect-pmm.md

```diff
@@ -5,9 +5,13 @@ Connect your PMM servers to Percona Platform to leverage Platform services that
 ## Pre-requisites
 To ensure that Percona Platform can establish a connection with the PMM instance:
 
-### Install PMM 2.25.0 or later
- Before connecting your PMM server to Percona Platform, make sure you are using PMM version 2.25 or newer. Otherwise, upgrade your PMM installation beforehand.
+### Install PMM 2.27.0 or later
+ Before connecting your PMM server to Percona Platform, make sure you are using PMM version 2.27 or newer. Otherwise, upgrade your PMM installation beforehand. 
  
+ Upgrading is required because PMM 2.25 and 2.26 used username/password authentication for connecting to Percona Platform. Starting with PMM 2.27, this was replaced with access token authentication to support federated identity and increase security. 
```

Comment:
```suggestion
 Upgrading is required because PMM 2.25 and 2.26 use username/password authentication for connecting to Percona Platform. Starting with PMM 2.27, this was replaced with access token authentication to support federated identity and increase security. 
```